src/Utilities/FeedLogger.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Pinterest for WooCommerce Feed Logger
+ *
+ * @package     Pinterest_For_WooCommerce/Classes/
+ * @since       x.x.x
+ */
+
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\Pinterest\Utilities;
+
+use Automattic\WooCommerce\Pinterest\Logger;
+
+/**
+ * Trait FeedLogger
+ *
+ * @since x.x.x
+ */
+trait FeedLogger {
```

Comment:
One minor thing: because the `$level` used for the logger is `product-sync`, I feel like this should be `ProductSyncLogger` rather than `FeedLogger`.

Reply:
Agree. Changed.